build.gradle

```diff
@@ -154,6 +154,17 @@ subprojects {
         }
     }
 
+    task generateProperties(type: WriteProperties) {
+        ext {
+            parsedVersion = VersionNumber.parse(version)
+        }
+        property("version.major", parsedVersion.getMajor())
```

Comment:
Since these are just java properties, should we namespace them a bit more? e.g. `org.conscrypt.version.major`?

Reply:
Sure, done.